scripts/release.sh

```diff
@@ -122,6 +154,51 @@ function parse_flags() {
     shift
   done
 
+  # Setup dot releases
+  if (( is_dot_release )); then
+    echo "Dot release requested"
+    TAG_RELEASE=1
+    PUBLISH_RELEASE=1
```

Comment:
do we allow not publishing for dot releases? What happens if I set --dot-release and --nopublish?

Reply:
Like any named releases, publishing is mandatory. This is explained in the documentation in each repo, and I intend to do the same about `--dot-release`. For example: https://github.com/knative/eventing/blob/master/hack/release.md#creating-versioned-releases